CHANGELOG.md

```diff
@@ -2,6 +2,18 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [4.0.2](https://github.com/paritytech/substrate-api-sidecar/compare/v4.0.1...v4.0.2) (2021-03-15)
+
+
+### Bug Fixes 
+
+* **types** Update @polkadot/api to get latest substrate types for `Extender{Header, SignedBlock}`, and updated metadata
+* **types** Update @polkadot/apps-config to get latest substrate types for `Extender{Header, SignedBlock}`, and updated metadata
```

Comment:
``
```suggestion
* **types** Update @polkadot/apps-config to get latest chain specific types
```

Reply:
The primary purpose of apps-config is chain specific type definitions; api is for substrate and polkadot types